DEVELOPMENT.md

```diff
@@ -0,0 +1,45 @@
+# Developing Delia
+
+Delia's stack aims to be straightforward and opinionated,
+using Typescript, Vite and Biome as the major development tools
+and React & Tailwind for the frontend due to their ubiquity.
+
+## Environment Setup
+
+Delia requires `pnpm`, which you can install in variety of ways:
```

Comment:
It's worth mentioning that without `collator` running somewhere/SPs you own't be able to make a use of it.